src/validateRegisterForm.test.js

```diff
@@ -27,4 +27,17 @@ describe(`Function 'validateRegisterForm':`, () => {
   });
 
   // write more tests here
+  it(`should return error for invalid email and valid passowd`, () => {
```

Comment:
There is a typo in the test description: 'passowd' should be corrected to 'password'.